Register RoIAlignRotated with C10

SuperIRabbit opened the pull request:
Differential Revision: D18415056

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit e3089a17:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=0392e8384bfbaaa2ad9f594fa7d95b8a06b5c146&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 0392e838 since Jan 15
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=e3089a17d958ea85f20dae46e5b87b5515712327).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_linux_xenial_py3_6_gcc5_4_build](https://dr.pytorch.org/build-details.html?build_id=86610144) from Jan 15



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

SuperIRabbit committed:
Register RoIAlignRotated with C10

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/30785

Differential Revision: D18415056

fbshipit-source-id: 2031554a2281093dcc3b5649f682736bd6155ca1
